scripts/setup-macos.sh

```diff
@@ -31,10 +31,7 @@ set -x # Print commands that are executed.
 SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
 source $SCRIPTDIR/setup-helper-functions.sh
 
-CPU_TARGET="${CPU_TARGET:-avx}"
-FB_OS_VERSION=v2022.07.11.00
 NPROC=$(getconf _NPROCESSORS_ONLN)
-COMPILER_FLAGS=$(get_cxx_flags $CPU_TARGET)
```

Comment:
whats the reason to remove COMPILER_FLAGS - they are required till we can migrate all the dependencies right ?